### PR TITLE
T5 279 drupalcon 2021 event page

### PR DESCRIPTION
**Pull request by naeluh**

### CODE REVIEW

File: src/pages/drupalcon-2021-event.js [hunk 1]
```diff
@@ -286,6 +122,8 @@ const Drupalicon = ({ data }) => {
         banner: true,
         styles: layoutStyles,
         navLink: joinLink,
+        logo: null,
+        heroLogo: true,
```
Comment (johnnybgoode) [inline src/pages/drupalcon-2021-event.js:126]: Logo images for TopNav should be set here if needed.

File: src/components/TopNav/TopNav.js [hunk 2]
```diff
@@ -57,6 +57,44 @@ const TopNav = ({ fill, hideNav, banner, navLink }) => {
...
+              }
+
+              .image {
+                width: 70px;
+              }
+            `,
+          ]}
+        >
+          <Img
+            alt='DrupalCon Silver Sponsor'
```
Comment (johnnybgoode) [inline src/components/TopNav/TopNav.js:86]: The `alt` text for this image should be passed in the page's `headerData` along with the logo image so the text can be changed along with the image if needed.

File: src/components/TopNav/TopNav.js [hunk 3]
```diff
@@ -203,7 +244,16 @@ const TopNav = ({ fill, hideNav, banner, navLink }) => {
         >
           <div className='top-bar--container'>
             <div className='left'>
-              <img src={DrupaliCon} alt='Drupalcon 2021' />
+              <Img
+                alt='DrupalCon Silver Sponsor'
```
Comment (johnnybgoode) [inline src/components/TopNav/TopNav.js:248]: The alt text for this image should be passed in the page's headerData along with the logo image so the text can be changed along with the image if needed. (same as above)

File: src/pages/drupalcon-2021-event.js (hunk 1 above)
Comment (naeluh) [reply src/pages/drupalcon-2021-event.js:124]: Good call on this, thanks, this actually ended up not being used. But yes you are correct. Thanks!

File: src/components/TopNav/TopNav.js (hunk 2 above)
Comment (naeluh) [reply src/components/TopNav/TopNav.js]: Yes you are correct this is how it should operate but it was removed from the design. Thanks!

File: src/components/TopNav/TopNav.js (hunk 3 above)
Comment (naeluh) [reply src/components/TopNav/TopNav.js]: Yep thanks!

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -327,6 +375,7 @@ export const headerPropTypes = {
   images: PropTypes.array,
   navLink: PropTypes.string,
   noIndex: PropTypes.bool,
+  heroLogo: PropTypes.bool,
```
Comment (johnnybgoode) [inline src/components/Header/Header.js:378]: Boolean variable names should start with `is` or `has` to show state and avoid confusion w/ variables that hold data 
e.g.
```
{
  hasHeroLogo: true,
  heroLogo: 'path/to/file.jpg'
}
```

File: src/components/Header/Header.js [hunk 5]
```diff
@@ -356,6 +405,7 @@ Header.defaultProps = {
   images: [],
   navLink: 'https://engage.acquia.com',
   noIndex: false,
+  heroLogo: false,
```
Comment (johnnybgoode) [inline src/components/Header/Header.js:408]: Match name change here.

File: src/components/Header/Header.js [hunk 6]
```diff
@@ -54,7 +63,11 @@ const Header = ({
   images,
   navLink,
   noIndex,
+  heroLogo,
```
Comment (johnnybgoode) [inline src/components/Header/Header.js:66]: Match name change here.

File: src/components/Header/Header.js [hunk 7]
```diff
@@ -264,6 +287,31 @@ const Header = ({
         height={height}
         minHeight={mobileMinHeight}
       >
+        {heroLogo && (
```
Comment (johnnybgoode) [inline src/components/Header/Header.js:290]: Match name change here.

File: src/components/Header/Header.js [hunk 8]
```diff
@@ -27,7 +35,8 @@ import FullWidthSection from '../FullWidthSection';
  * @param {string} image - passed to SEO
  * @param {string} heroImage - used as background on desktop
  * @param {string} heroImageMobile - used as background on mobile
- * @param {bool} noIndex
+ * @param {bool} noIndex - set noindex,nofollow to page
+ * @param {string} heroLogo - optional logo in hero area
```
Comment (johnnybgoode) [inline src/components/Header/Header.js:39]: Match name change here.

Comment (johnnybgoode) [reply src/components/Header/Header.js:39]: Update type to `{bool}` and update description

File: src/components/Header/Header.js (hunk 4 above)
Comment (naeluh) [reply src/components/Header/Header.js:378]: Yep I added an alt text as well, thanks!

File: src/components/Header/Header.js (hunk 5 above)
Comment (naeluh) [reply src/components/Header/Header.js:408]: Updated thanks!

File: src/components/Header/Header.js (hunk 6 above)
Comment (naeluh) [reply src/components/Header/Header.js:66]: Updated thanks!

File: src/components/Header/Header.js (hunk 7 above)
Comment (naeluh) [reply src/components/Header/Header.js:290]: Updated thanks!

File: src/components/Header/Header.js (hunk 8 above)
Comment (naeluh) [reply src/components/Header/Header.js:39]: Updated thanks!

Comment (naeluh) [reply src/components/Header/Header.js:41]: Updated thanks!

**Review by johnnybgoode — APPROVED** [review]

**Pull request merged**